pkg/test/expectations.go

```diff
@@ -0,0 +1,46 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"encoding/base64"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/service/ec2"
+	. "github.com/onsi/gomega" // nolint:revive,stylecheck
+)
+
+func (env Environment) ExpectLaunchTemplatesCreatedWithUserDataContaining(substrings ...string) {
```

Comment:
nit: Are these functions that need to be in the generic test/expectations file or can they be scoped to the launchtemplate/suite_test?